scss/civicrm/common/_modals.scss

```diff
@@ -2,6 +2,7 @@
 @import '../bootstrap/overrides/style/modals';
 
 .crm-container.ui-dialog {
+  $ui-icon-margin: 7px;
```

Comment:
Should not that go to variables?

Reply:
I think this is too localized to be part of the general variables.